compiler/damlc/daml-stdlib-src/DA/Internal/Prelude.daml

```diff
@@ -26,6 +26,8 @@ import GHC.Tuple()
 import GHC.Types as GHC (Bool (..), Int, Ordering (..), Text, Decimal, ifThenElse, primitive)
 
 infixr 0 $
+-- | Take a function from `a` to `b` and a value of type `a`, and apply the
+-- function to `a`, returning a `b`.
```

Comment:
If you want to be precise, we are applying the function to the value of type `a` not to `a`. It might also be worth pointing out that the point of this function is that it has a very low precedence, otherwise it’s unclear why you would want to use this as opposed to regular function application.